Python/ceval.c

```diff
@@ -2124,6 +2124,16 @@ _PyEval_UnpackIterableStackRef(PyThreadState *tstate, _PyStackRef v_stackref,
         _PyErr_Format(tstate, PyExc_ValueError,
                       "too many values to unpack (expected %d)",
                       argcnt);
+        PyObject *exc = _PyErr_GetRaisedException(tstate);
+        PyObject_SetAttr(exc,
+                         PyUnicode_FromString("_is_incomplete_unpack_errmsg"),
+                         Py_True);
+        PyObject_SetAttr(exc, PyUnicode_FromString("_unpacked_value"), v);
+        PyObject_SetAttr(exc,
```

Comment:
Each SetAttr can fail. We know the type here (`ValueError`), but we could still get something like a `MemoryError` that should bubble up.

---

`PyUnicode_FromString` can fail in the same way.
But, CPython has a special internal way to deal with compile-time constant strings:
- use `&_Py_STR(_unpacked_value)`
- run `make regen-global-objects` to find the `_Py_STR` and (re)create the code for it. (If you're on Windows, use a container, or run `./Tools/build/generate_global_objects.py` directly.)